packages/components/src/components/primitives/Accordion/Accordion.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { FC } from 'react';
+import {
+  Accordion as ChakraAccordion,
+  AccordionItem,
+  AccordionButton as ChakraAccordionButton,
+  AccordionPanel,
+  AccordionIcon,
+} from '@chakra-ui/react';
+
+import {
+  AccordionProps,
+  AccordionButtonProps,
+  AccordionStaticMembers,
+} from './types';
+
+export const Accordion: FC<AccordionProps> & AccordionStaticMembers = (
+  props
+) => {
+  return <ChakraAccordion _expanded={{ bg: '#]]]' }} {...props} />;
+};
+
+const AccordionButton: FC<AccordionButtonProps> = ({ highlight, ...props }) => {
+  let expanded = null;
+  if (highlight) {
+    expanded = { bg: 'background-action-active' };
```

Comment:
Could we simplify this to something like this so we can remove the expanded variable?

`  {...(highlight && { bg: 'background-action-active' })}`

Reply:
done!